tests/api_tests/nodes/test_views.py

```diff
@@ -1352,7 +1467,7 @@ def test_filtering_on_invalid_field(self):
         assert_equal(res.status_code, 400)
         errors = res.json['errors']
         assert_equal(len(errors), 1)
-        assert_equal(errors[0]['detail'], 'Query string contains an invalid filter.')
+        assert_equal(errors[0]['detail'], "'invalid' is not a valid field for this endpoint.")
```

Comment:
For security reasons, we've avoided including user input in our error messages. While it should be the client's responsibility to clean strings, I think it's most prudent to avoid the problem altogether.